frontend/src/app/main/data/workspace/colors.cljs

```diff
@@ -390,3 +392,26 @@
     (update [_ state]
       (-> state
           (assoc-in [:workspace-global :editing-stop] spot)))))
+
+(defn manage-color-shapes
```

Comment:
Perhaps move this function inside the event, instead of being a global function. And the name could be `select-shapes-for-color` or similar.

Reply:
Ok, I see. Let me know if now it's better